lexoid/core/parse_type/static_parser.py

```diff
@@ -1,11 +1,12 @@
-from typing import Dict, List
-
+import tempfile
 import pandas as pd
 import pdfplumber
 import pymupdf4llm
+from typing import List, Dict
+from lexoid.core.utils import get_uri_rect, split_pdf
 from pdfminer.high_level import extract_pages
 from pdfminer.layout import LTTextContainer
-from pdfplumber.utils import extract_text, get_bbox_overlap, obj_to_bbox
+from pdfplumber.utils import get_bbox_overlap, obj_to_bbox
 
 
 def parse_static_doc(path: str, raw: bool, **kwargs) -> List[Dict] | str:
```

Comment:
Should we change the default framework in `parse_static_doc`?

Reply:
Done